src/index.js

```diff
@@ -1,46 +1,162 @@
+    const dustbin = document
+      .querySelector('#list')
+      .querySelectorAll('.fa-trash-can');
+
+    dustbin.forEach((bin) => {
+      bin.addEventListener('click', (e) => {
+        // Remove item from UserInterface
+        UserInterface.removeToDoLists(e.target.parentElement);
+        // Remove item from LocalStorage
+        LocalStorage.removeToDoLists(
+          e.target.parentElement.previousElementSibling.children[1].value,
+        );
+      });
+    });
+  }
+
+  static addToDoLists(item) {
+    const listContent = document.querySelector('#list');
     const listElt = document.createElement('li');
-    listElt.dataset.listId = i.index;
-    listElt.classList.add('checkbox');
-    listElt.innerHTML += `<span><input type="checkbox" /> ${i.description}</span><img src="${dots}" />`;
-    enterImg.setAttribute('src', enter);
-    refreshImg.setAttribute('src', refresh);
-    listContainer.appendChild(listElt);
-  });
-};
-
-showList();
+    listElt.classList = 'new';
+    listElt.id = `${item.index}`;
+    listElt.innerHTML += `
+      <span class="inputs"><input class="checkbox" type="checkbox" /><input class="desc" type="text" value="${item.desc}" /></span>
+      <button class="text-btn" type="button">
+        <i class="fa-solid fa-trash-can"></i>
+      </button>
+      `;
+    listContent.appendChild(listElt);
+    // <i class="fa-solid fa-ellipsis-vertical" name="update"></i>
+    const listContainer = document.querySelector('.form-text');
+    listContainer.innerHTML = `
+      <p class="error-message">*Error</p>
+      <form class="form-text" action="">
+        <input type="text" class="text" placeholder="Add to your list..." />
+        <i class="fa-solid fa-arrow-right-to-bracket"></i>
+      </form>
+    `;
+  }
+
+  static removeToDoLists(elt) {
+    if (elt.classList.contains('text-btn')) {
+      elt.parentElement.remove();
+    }
+    localStorage.setItem('todolists', JSON.stringify(todolists));
+  }
+
+  static clearFields() {
+    document.querySelector('.text').value = '';
+  }
+}
+
+// DISPLAY TO DO LIST
+document.addEventListener('DOMContentLoaded', UserInterface.showToDoLists);
+
+// ADD TO DO LIST
+document.querySelector('.form-text').addEventListener('submit', (e) => {
+  // Prevent actual submit
+  e.preventDefault();
+
+  // Error Handling
+  const inputText = document.querySelector('.text');
+  const errorMsg = document.querySelector('.error-message');
+  if (inputText.value == null || inputText.value === '') {
+    errorMsg.setAttribute('style', 'display:block');
+    return;
+  }
+  errorMsg.setAttribute('style', 'display:none');
+
+  // const index = Array.from(e.target.parentNode.parentNode.children).indexOf(
+  //   e.target.parentNode
+  // );
+  // console.log(e.target.children[2]);
+  // console.log(e.target.children[2].childElementCount);
+  // console.log(Array.from(e.target.children).indexOf(e.target.children[2]));
+  // console.log(todolists[todolists.length - 1].index);
```

Comment:
- [ ] Please do not push commented code to the repo. Comments render your codebase messy.